services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RestStop.java

```diff
@@ -93,6 +101,13 @@ public abstract static class Builder extends DirectionsJsonObject.Builder<Builde
      */
     public abstract Builder name(@Nullable String name);
 
+    /**
+     * The list of amenities at the rest stop if available.
+     *
+     * @param amenities list of amenities
+     */
+    public abstract Builder amenities(@Nullable List<Amenities> amenities);
```

Comment:
```suggestion
     */
    @NonNull 
    public abstract Builder amenities(@Nullable List<Amenities> amenities);
```

We can also add the above missing annotations 👍